rpcs3/Emu/Cell/Modules/cellVdec.cpp

```diff
@@ -84,7 +85,7 @@ struct vdec_context final
 	u64 next_dts{};
 	u64 ppu_tid{};
 
-	std::queue<vdec_frame> out;
+	std::deque<vdec_frame> out;
```

Comment:
I don't get why you changed it to deque.

Reply:
for iteration support.